layouts/docs/glossary.html

```diff
@@ -5,12 +5,12 @@
         </ul>
     </div>
 {{ end }}
-    
+
 {{ range sort $.Site.Data.glossary.entries "term" }}
     <div class="flex-1 mt-8">
         <h2 class="mt-2 mb-0">{{ .term }} </h2>
             <p>{{ .description }}</p>
                 <ul class="mb-4">
-                    <li>Read More: <a href="{{ .link }}" class="">{{ .term }}</a></li>
+                    <li>Read More: <a data-track="read-more-{{ .term | urlize }}-read-more" href="{{ .link }}" class="">{{ .term }}</a></li>
```

Comment:
Just curious: Why is there both a `read-more-` prefix and `-read-more` suffix in the `data-track` value?

Reply:
Because I'm an idiot and didn't realize I put it on both sides lol. Fixing.